v2/models/autoevent.go

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package models
+
+// AutoEvent supports auto-generated events sourced from a device service
+// This object and its properties correspond to the AutoEvent object in the APIv2 specification:
+// https://app.swaggerhub.com/apis-docs/EdgeXFoundry1/core-metadata/2.x#/AutoEvent
```

Comment:
I don't think it is validate to have this link here as the Models are not part of the Swagger. This should only be present on the DTOs. Please remove from all model files.

Reply:
Removed from all models.